presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -794,6 +795,11 @@ protected Type visitFunctionCall(FunctionCall node, StackableAstVisitorContext<C
                                         symbolTypes,
                                         parameters,
                                         isDescribe);
+                                if (context.getContext().isInLambda()) {
+                                    for (LambdaArgumentDeclaration lambdaArgumentDeclaration : context.getContext().getNameToLambdaArgumentDeclarationMap().values()) {
```

Comment:
Just call this `argument` or something shorter? The verbosity in the name only makes the code harder to read.